packages/extensions/src/ext-meshopt-compression/encoder.ts

```diff
@@ -73,20 +74,18 @@ export function getMeshoptFilter(accessor: Accessor, doc: Document): MeshoptFilt
 		.map((link) => (link as AttributeLink).getName())
 		.filter((name) => name !== 'accessor');
 	for (const semantic of semantics) {
-		// TODO(filter): Need to pad the normals with a .w component, I think?
-		if (semantic === 'NORMAL') return MeshoptFilter.NONE;
+		if (semantic === 'NORMAL') return MeshoptFilter.OCTAHEDRAL;
 		if (semantic === 'TANGENT') return MeshoptFilter.OCTAHEDRAL;
 		if (semantic.startsWith('JOINTS_')) return MeshoptFilter.NONE;
-		if (semantic.startsWith('WEIGHTS_')) return MeshoptFilter.NONE;
+		if (semantic.startsWith('WEIGHTS_')) return MeshoptFilter.EXPONENTIAL;
```

Comment:
I'd recommend 8-bit quantization (make sure the weights add up to 255 exactly) and no filter instead here.

Reply:
Thanks!